### ISSUE TITLE
Who can use this code to train the model, which can get the same results as author's? I get loss:nan.

### ISSUE THREAD

**Issue by hdjsjyl** [Goingqs/PyramidBox#10]
Who use this code to train the model, which can get the same results as author's? I get loss:nan.

**Comment by hdjsjyl** [comment]
I can not train the network successfully.

**Comment by pbdahzou** [comment]
try a smaller learning-rate

**Comment by hdjsjyl** [comment]
@pbdahzou  I change learning-rate to 0.0005. Do you think I need to change other parameters? Thanks

**Comment by pbdahzou** [comment]
I think you can firstly try the smaller learning-rate and see whether it works or not.

**Comment by hdjsjyl** [comment]
@pbdahzou  Yes, until now it works and it has been trained 5330 images.

**Comment by pbdahzou** [comment]
what's your batch_size?  when I set 6 it already occupies 11G GPU :( .   Do you have a QQ?

**Comment by hdjsjyl** [comment]
@pbdahzou  yes, my qq is xxxxxxx

**Comment by pbdahzou** [comment]
@hdjsjyl I have added your QQ

**Comment by hdjsjyl** [comment]
@pbdahzou  I donot receive the message. I can add your QQ.

**Comment by pbdahzou** [comment]
@hdjsjyl my qq: xxxxx

**Comment by Edwardmark** [comment]
@pbdahzou Have you trained the model successfully?

**Comment by pbdahzou** [comment]
@Edwardmark yes

**Comment by pbdahzou** [comment]
@Edwardmark you can try a small learning rate such as 0.0001.

**Comment by hdjsjyl** [comment]
@pbdahzou  Can you share your parameters setting? what is the lr, what is the number of iterations, what is the stepvalues, what is the batchsize? Does your model get the same performance as author's? Thanks

**Comment by Edwardmark** [comment]
@pbdahzou I try 1e-4, but the result is not good.   What is your training log when converged? Could you please share with us?
```
[30620] Wed Sep 12 01:36:01 2018, lr: 0.0001, Loss: 6.750296592712402, Loss conf: 1.9768489599227905, Loss loc: 2.557082176208496, Loss head conf: 1.9376943111419678, Loss head loc: 2.4950361251831055
```

**Comment by Edwardmark** [comment]
@hdjsjyl What is your training result tested on WIDERFACE?

**Comment by Edwardmark** [comment]
@pbdahzou What is your training log and the model tested on WIDER_FACE?Would you please give us some help? Can you provide a training list?

**Comment by YueXiNPU** [comment]
> 
> 
> @hdjsjyl my qq: xxxxx

hello, I am running Pyramidbox. I want to talk to you. What's your email address?

**Comment by abcdvzz** [comment]
> > @hdjsjyl my qq: xxxxx
> 
> hello, I am running Pyramidbox. I want to talk to you. What's your email address?

What about your training now?

**Comment by abcdvzz** [comment]
@Edwardmark What about your training now?

**Comment by Edwardmark** [comment]
@abcdvzz I finally can train it.

**Comment by abcdvzz** [comment]
> @abcdvzz I finally can train it.

Could u pls public your code? I cannot make it.

**Comment by Tianlock** [comment]
@Edwardmark  My qq is 468651841,could you add me and i want ask you some question, thanks.
